### DISCUSSION TITLE
How to parse AuxVariables to a vector?

### DISCUSSION THREAD

**Discussion by Minjiang-Zhu** [idaholab/moose#22753] [Q&A General] [answered]
As described above.

**Comment by GiudGiud** [comment]
Hello

what do you mean?
Use it in a parsed expression? That’s possible for the ParsedMaterial or the ParsedAux

guillaum

**Comment by Minjiang-Zhu** [comment, marked as answer]
I use a vector function and apply ParsedVectorFunction to fix it.

**Reply by GiudGiud** [reply]
ok I saw the other post. Please consolidate on a single post in the future.
